docs/Displaying a User Interface/Main Screen Templates/index.md

```diff
@@ -134,6 +134,255 @@ if (success) {
 ```
 !@
 
+@![iOS]
+When changing screen layouts or template fields, it is highly recommended to encapsulate these updates into a class or method. Doing so is a good way to keep SDL UI changes organized. An example of this can be seen in the [example weather app](https://github.com/SmartDeviceLink-Examples/example_weather_app_ios). See below for another example:
+
+|~
+```objc
+// CustomSDLScreen.h - Protcol
+@class SDLManager;
+
+@protocol CustomSDLScreen <NSObject>
+
+@required
+- (instancetype)initWithManager:(SDLManager *)sdlManager;
+- (void)showScreen;
+
+@end
+
+// HomeSDLScreen.h
+#import <Foundation/Foundation.h>
+#import "CustomSDLScreen.h"
+
+@class HomeDataModel;
+@class SDLManager;
+@class SDLScreenManager;
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface HomeSDLScreen : NSObject<CustomSDLScreen>
+
+@end
+
+NS_ASSUME_NONNULL_END
+
+// HomeSDLScreen.m
+#import "HomeSDLScreen.h"
+
+#import "CustomSDLScreen.h"
+#import "HomeDataModel.h"
+#import "SmartDeviceLink.h"
+
+@interface HomeSDLScreen()
+
+@property (strong, nonatomic) SDLManager *sdlManager;
+@property (assign, nonatomic) HomeDataModel *homeDataModel;
```

Comment:
```suggestion
// An example of your data model that will feed data to the SDL screen's UI
@property (strong, nonatomic) HomeDataViewModel *homeDataViewModel;
```

The way you're using this is really more like a view model than a model. Please rename it to reflect that.